### PR TITLE
Update status of proposals

### PR DESCRIPTION
**Pull request by jvican**
This PR goes through outdated proposals and update them with their latest status. Review by @SethTisue | @martijnhoekstra.

### CODE REVIEW

File: sips/completed/_posts/2012-03-17-modularizing-language-features.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: sip
 title: SIP-18 - Modularizing Language Features
 
 vote-status: accepted
-vote-text: This is in Accept status, pending implementation. Let the record show Paul is against it.
+vote-text: This SIP has already been accepted with the vote against of Paul.
```
Comment (heathermiller) [inline sips/completed/_posts/2012-03-17-modularizing-language-features.md]: I prefer the original wording. Why did this need to be updated?

File: sips/pending/_posts/2013-05-31-improved-lazy-val-initialization.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ disqus: true
 title: SIP-20 - Improved Lazy Vals Initialization
 
 vote-status: under review
-vote-text: Next iteration takes place in October 2016.
+vote-text: Dormant. This proposal lacks an implementation for Scalac and is looking for a new author.
```
Comment (heathermiller) [inline sips/pending/_posts/2013-05-31-improved-lazy-val-initialization.md]: Looking for a new *owner*, not author.

File: sips/pending/_posts/2013-06-30-async.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ disqus: true
 title: SIP-22 - Async
 
 vote-status: postponed
-vote-text: Authors have asked to postpone until further notice. They will eventually open the proposal when some details in the implementation are ready. Check <a href="http://localhost:4000/sips/minutes/sip-minutes.html">July's minutes</a>.
+vote-text: Authors have asked to postpone this proposal until further notice. They will eventually open the proposal when some details in the implementation are ready. Check <a href="http://localhost:4000/sips/minutes/sip-minutes.html">July's minutes</a>.
```
Comment (heathermiller) [inline sips/pending/_posts/2013-06-30-async.md]: Isn't this dormant? They don't plan on picking it up.

File: sips/pending/_posts/2014-06-27-42.type.md [hunk 4]
```diff
@@ -2,6 +2,9 @@
 layout: sip
 disqus: true
 title: SIP-23 - Literal-based singleton types
+
+vote-status: dormant
+vote-text: This proposal is dormant because original authors are not available for its review. There is an implementation <a href="https://github.com/scala/scala/pull/5310">here</a>.
```
Comment (heathermiller) [inline sips/pending/_posts/2014-06-27-42.type.md]: Did you test this locally? I was under the impression that links don't work in front-matter.

File: sips/completed/_posts/2016-06-25-trailing-commas.md [hunk 5]
```diff
@@ -2,9 +2,10 @@
 layout: sip
 disqus: true
 title: SIP-27 - Trailing Commas
+redirect_from: "/sips/pending/trailing-commas.html"
 
-vote-status: under review
-vote-text: The following proposal needs to be updated, since only the specialized case version (with new lines) has been accepted. For more information, check the <a href="http://docs.scala-lang.org/sips/minutes/sip-20th-september-minutes.html">minutes</a>.
+vote-status: approved
+vote-text: The following proposal has been approved but the spec needs to be updated and the implementation has to be ported from Parsers to Scanner. For more information, check the minutes in November 2017.
```
Comment (heathermiller) [inline sips/completed/_posts/2016-06-25-trailing-commas.md]: November 2017? Do you mean November 2016? In any case, these minutes do not exist yet, so it's impossible to check this. Perhaps omit this until you have done the minutes?

File: sips/pending/_posts/2015-6-18-trait-parameters.md [hunk 6]
```diff
@@ -2,6 +2,9 @@
 layout: sip
 title: SIP 25 - Trait Parameters
 disqus: true
+
+vote-status: under review
+vote-text: The board agreed to schedule the next iteration of the evaluation process in 6 months, since there’s no implementation yet and the authors need time to produce one.
```
Comment (heathermiller) [inline sips/pending/_posts/2015-6-18-trait-parameters.md]: I was under the impression that this was dormant and needs a new owner. Martin was not going to implement this.

File: sips/pending/_posts/2015-6-18-repeated-byname.md [hunk 7]
```diff
@@ -2,6 +2,9 @@
 layout: sip
 title: SIP-24 - Repeated By Name Parameters
 disqus: true
+
+vote-status: dormant
+vote-text: Looking for a new author. This proposal needs to be updated according to the <a href="https://www.youtube.com/watch?v=hrxp4PGbi6s">SIP meeting in November 2016</a>.
```
Comment (heathermiller) [inline sips/pending/_posts/2015-6-18-repeated-byname.md]: Owner, not author.

File: sips/pending/_posts/2014-06-27-42.type.md (hunk 4 above)
Comment (heathermiller) [inline sips/pending/_posts/2014-06-27-42.type.md]: Again, owner, not author.

File: sips/pending/_posts/2013-05-31-improved-lazy-val-initialization.md (hunk 2 above)
Comment (jvican) [inline sips/pending/_posts/2013-05-31-improved-lazy-val-initialization.md]: 👍

File: sips/completed/_posts/2012-03-17-modularizing-language-features.md (hunk 1 above)
Comment (jvican) [inline sips/completed/_posts/2012-03-17-modularizing-language-features.md]: I just wanted to emphasize that it was accepted, meaning that it's been already merged into scalac. Will change.

File: sips/pending/_posts/2013-06-30-async.md (hunk 3 above)
Comment (jvican) [inline sips/pending/_posts/2013-06-30-async.md]: Yeah, dormant is more appropriate in this case.

File: sips/completed/_posts/2016-06-25-trailing-commas.md (hunk 5 above)
Comment (jvican) [inline sips/completed/_posts/2016-06-25-trailing-commas.md]: Yes, I will omit it for now. Thanks for the catch in the date.

**Comment by SethTisue** [comment]
@jvican should we go ahead and merge?

**Comment by SethTisue** [comment]
@jvican could we get this across the finish line...?

**Comment by jvican** [comment]
Hey @SethTisue, updating tomorrow! Sorry this fell through the cracks.

**Comment by jvican** [comment]
@heathermiller This is ready for review again 😄

**Comment by heathermiller** [comment]
Thanks @jvican and @SethTisue!! ❤️

**Pull request merged**